.appveyor.yml

```diff
@@ -0,0 +1,64 @@
+build: false
+platform:
+  - x64
+clone_folder: C:\projects\migrations
+clone_depth: 1
+
+cache:
+  - C:\ProgramData\chocolatey\bin -> .appveyor.yml
+  - C:\ProgramData\chocolatey\lib -> .appveyor.yml
+  - C:\tools\php -> .appveyor.yml
+  - C:\tools\composer -> .appveyor.yml
+  - '%LOCALAPPDATA%\Composer\files -> composer.json'
+
+environment:
+  matrix:
+    - php: 7.2
+
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;C:\tools\composer;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET ANSICON=121x90 (121x90)
+
+## Install PHP and composer, and run the appropriate composer command
+install:
+    - ps: |
+        # Check if installation is cached
+        if (!(Test-Path c:\tools\php)) {
+          appveyor-retry cinst --params '""/InstallDir:C:\tools\php""' --ignore-checksums -y php --version ((choco search php --exact --all-versions -r | select-string -pattern $env:php | sort { [version]($_ -split '\|' | select -last 1) } -Descending | Select-Object -first 1) -replace '[php|]','')
+          # install sqlite
+          appveyor-retry cinst -y sqlite
+          Get-ChildItem -Path c:\tools\php
+          cd c:\tools\php
+
+          # Set PHP environment items that are always needed
+          copy php.ini-production php.ini
```

Comment:
@Majkl578 can you tell me where to find this php.ini-production file?

Reply:
It's part of php-src itself.